component/core/src/test/java/org/exoplatform/social/core/jpa/storage/dao/SpaceDAOTest.java

```diff
@@ -100,7 +100,7 @@ private SpaceEntity createSpace(String spacePrettyName) {
     spaceEntity.setVisibility(SpaceEntity.VISIBILITY.PRIVATE);
     spaceEntity.setBannerLastUpdated(new Date());
 
-    addMember(spaceEntity, "root", SpaceMemberEntity.Status.PENDING);
+    addMember(spaceEntity, "root", SpaceMemberEntity.Status.MEMBER);
```

Comment:
You should not change that. You will affect other tests.
Create a new function if needed